mkdocs.yml

```diff
@@ -162,3 +174,5 @@ extra:
   analytics:
     provider: google
     property: G-JXQ1R227VS
+#  navigation: 
```

Comment:
I think this would need to go under the `theme: features:` keys (like [here](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion)).

But it looks like it doesn't support scoping to specific sections, so it's sitewide or none :(